docs/src/pages/components/slider/slider.md

```diff
@@ -48,6 +48,22 @@ Continuous sliders allow users to select a value along a subjective range.
 
 {{"demo": "pages/components/slider/VerticalSlider.js"}}
 
+## Track
+
+The track shows the range available for user selection.
+
+### Removed track
+
+The track can be turned off with `track={false}`.
```

Comment:
Nitpicking, but isn't the track the underlying element, that is the full length of the slider? The spec doesn't seem to have specific terminology for the part indicating the selected range though.

Reply:
I believe we call it the "rail".

Reply:
"Track" and "rail" sound the same to me...

Reply:
This sounds about right. We can rename it in v5. Any idea?

Reply:
I have added a note about in the list of BC potentials for v5. At least, we won't forget about it.